packages/stream_feed/test/mock.dart

```diff
@@ -27,21 +27,21 @@ class MockDio extends Mock implements Dio {
 
 class MockHttpClient extends Mock implements StreamHttpClient {}
 
-class MockAnalyticsApi extends Mock implements AnalyticsApi {}
+class MockAnalyticsAPI extends Mock implements AnalyticsAPI {}
 
-class MockCollectionsApi extends Mock implements CollectionsApi {}
+class MockCollectionsAPI extends Mock implements CollectionsAPI {}
 
-class MockFilesApi extends Mock implements FilesApi {}
+class MockFilesAPI extends Mock implements FilesAPI {}
 
-class MockReactionsApi extends Mock implements ReactionsApi {}
+class MockReactionsAPI extends Mock implements ReactionsAPI {}
 
-class MockImagesApi extends Mock implements ImagesApi {}
+class MockImagesAPI extends Mock implements ImagesAPI {}
 
-class MockUsersApi extends Mock implements UsersApi {}
+class MockUsersApi extends Mock implements UsersAPI {}
```

Comment:
MockUsersAPI

Reply:
```suggestion
class MockUsersAPI extends Mock implements UsersAPI {}
```

Might also need to refactor wherever it is used.